.gitignore

```diff
@@ -1,6 +1,7 @@
 /docker/.data
 /cli/target/
 /webapp/target/
+/webapp/package-lock.json
```

Comment:
@aurambaj We don't need this file to be checked into git, correct?

Reply:
yes it is needed it git, don't exclude it please.

Reply:
Okay. I have reverted deletion of package-lock.json changes now.